src/api/itemusages.go

```diff
@@ -16,6 +16,7 @@ type ItemUsage struct {
 	ItemUUID    string          `json:"item_uuid"`
 	User        ItemUsageUser   `json:"user"`
 	Client      ItemUsageClient `json:"client"`
+	Action      string          `json:"action,omitempty"`
```

Comment:
Do we need to map this field to a CIM field? (@hilaryhacksel?)

Reply:
These mappings are kind of difficult because splunk's models doesn't fit our use case very well. For item usages we tried to alias our events to splunk's [Change model](https://docs.splunk.com/Documentation/CIM/4.19.0/User/Change) since it was the closest in shape and intention. I'm seeing there that there is an action listed in this model, as Anthony pointed out. However, I think what might be more accurate is the `command` field. If you all agree, I would alias our `action` to Splunk's `command`.